src/dtw.jl

```diff
@@ -49,6 +56,59 @@ function dtw_bhattacharyya(seq1::Array{kmerdist}, seq2::Array{kmerdist})
     return trackback_bhattacharyya(D)
 end
 
+function dtw_multiple_cost_matrix_bhattacharyya(seq::Array{Array{Float32}},
+    segmentsize = 100, transportcost=1) where T
```

Comment:
you are defining T will come later during function call, but where is T being actually used inside the function?